{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py

```diff
@@ -26,5 +26,5 @@ class Meta(object):  # pylint:disable=missing-docstring
     def get_full_name(self):
         return self.full_name or super(User, self).get_full_name()
 
-    def __unicode__(self):
-        return unicode(self.get_full_name())
+    def __str__(self):
```

Comment:
This should use the [python_2_unicode_compatible](https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible) decorator so it works cleanly under all supported Python versions